server/app.js

```diff
@@ -40,10 +43,31 @@ app.get('/search', function(req, res) {
   });
 });
 
+app.post('/users/signup', urlencodedParser, function(req, res) {
+  console.log(req.body);
+  var user = req.body //need to tailor this
+  //check if user exists
+  db.query('SELECT username from Users WHERE username = ' + req.body.username + '', function(err , result ) {
```

Comment:
this would a good spot to create a helper function in database.js then call that function, a promise wouldn't be bad either